modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/config/LazyInitExpressionConfigRefTestCase.java

```diff
@@ -34,20 +32,6 @@ public boolean enableLazyInit() {
     return true;
   }
 
-  @Ignore("W-11525405: remove ignore override once we support lazy initialization of configurations referenced by expressions")
-  @Test
-  @Override
-  public void getPetsWithExpression() throws Exception {
-    super.getPetsWithExpression();
-  }
-
-  @Ignore("W-11525405: remove ignore override once we support lazy initialization of configurations referenced by expressions")
-  @Test
-  @Override
-  public void getPetsWithExpressionResolvingToIncompatibleConfigFails() throws Exception {
-    super.getPetsWithExpressionResolvingToIncompatibleConfigFails();
-  }
-
```

Comment:
It looks like we are removing tests but we are actually unignoring inherited tests.